packages/modal/src/modal-transition.tsx

```diff
@@ -30,6 +31,7 @@ export const ModalTransition = React.forwardRef(
   (props: ModalTransitionProps, ref: React.Ref<any>) => {
     const { preset, ...rest } = props
     const motionProps = transitions[preset]
+    console.log(motionProps)
```

Comment:
Please remove this logging statement

Reply:
Thank you for the review! I will  remove the logging statement and add the new motion preset information in the docs.